graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/user/GqlUser.java

```diff
@@ -86,11 +83,56 @@ public GqlUser(JahiaUser jahiaUser) {
 
     @GraphQLField
     @GraphQLNonNull
+    @GraphQLDeprecate
```

Comment:
you can remove the field if needed, it has never been released anyway :-)

Reply:
This User object is also used by `currentUser` query so I didn't remove it in case the current user is used elsewhere, if it's safe from your point of view I can remove it

Reply:
i did not realize it was already there, so let's keep it